agilex_piper/piper.xml

```diff
@@ -0,0 +1,300 @@
+<mujoco model="piper_description">
+  <compiler angle="radian" meshdir="assets/" texturedir="assets/" />
+
+  <default>
+    <default class="visual">
+      <geom group="2" type="mesh" contype="0" conaffinity="0" density="0" />
+    </default>
+    <default class="collision">
+      <geom group="3" type="mesh" />
+    </default>
+    <joint damping="0.1" stiffness="1" frictionloss="0.1" armature="0.1" />
+    <general gaintype="fixed" biastype="affine" gainprm="2000" biasprm="0 -2000 -200" />
```

Comment:
Let's switch to position actuators for more clarity.

`<position kp="100" dampratio="1" inheritrange="1">`

I lowered the gains so please add gravcomp="1" to all the bodies.